lib/api/1.1/northbound/router.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+var di = require('di'),
+    express = require('express');
+
+module.exports = northApiFactory;
+
+di.annotate(northApiFactory, new di.Provide('northbound-api-router'));
```

Comment:
I see you put this file under folder `/api/1.1` , if we have both 1.1 and 2.0 API coexists in future, this injectable name will have conflict, so then we may have to introduce the version number in the injectable name.